terraform/modules/bap_env_gcp/disk.tf

```diff
@@ -0,0 +1,147 @@
+resource "google_compute_disk" "mariadb" {
+  count    = var.disk_mariadb_count
+  name     = "${var.prefix}-mariadb-disk-${count.index}"
+  type     = var.disk_mariadb_type
+  size     = var.disk_mariadb_size
+  image    = var.disk_mariadb_image
+  snapshot = var.disk_mariadb_snapshot
+  zone     = var.zone
+
+}
+
+resource "google_compute_attached_disk" "mariadb" {
+  count    = var.disk_mariadb_count
+  disk     = google_compute_disk.mariadb[count.index].id
+  instance = var.disk_mariadb_attach
```

Comment:
What is this?

Reply:
This resource is to attach the disk to an instance, but I'm going to change it so that by default it will attach to the same VM name like `test-mariadb-disk-0` attach to `test-mariadb-0`